source/modules/MEI.header.xml

```diff
@@ -2309,6 +2311,7 @@
     <classes>
       <memberOf key="att.common"/>
       <memberOf key="att.bibl"/>
+      <memberOf key="att.docStatus"/>
```

Comment:
Remove `<revisiondesc>` membership in `att.docStatus` since the current status of a document can be determined from the chronologically last `<change>` element.

Reply:
Thank you @pe-ro , we discussed this at the ODD-Meeting last week. There, we also pointed out to remove it here.